server/store/middleware/client/whitelist.go

```diff
@@ -14,16 +15,22 @@ func init() {
 	tracker.RegisterAnnounceMiddleware("client_whitelist", whitelistAnnounceClient)
 }
 
+// PrefixClient is the prefix to be used for client peer IDs.
+const PrefixClient = "pi-"
```

Comment:
I'd change this to be `c-` and the godoc to use client IDs instead of client peer IDs.